.github/workflows/build_deploy.yml

```diff
@@ -0,0 +1,74 @@
+name: Publish Docker image and deploy it
+
+on:
+  push:
+    branches:
+      - develop
+      - main
+
+env:
+  image_name: lucyparsons/openoversight
+  # env_mapping: {"develop": "staging", "main": "prod"}
```

Comment:
If these bits are gonna be commented out, I'd be game for just deleting them from the file.

Reply:
I had preferred to keep this in the code (not put it in a variable) but I somehow couldn't make that work. I left this here so it's easier to for people to make sense of it, even if they cannot see the variable settings (which might require admin rights to see, not sure)